crates/http-sink/src/sink.rs

```diff
@@ -39,11 +39,12 @@ impl Sink<String> for HttpSink {
                 trace!("{:?}", request);
 
                 request = request.body(record);
-                request
+                let response = request
                     .try_clone()
                     .ok_or(anyhow!("ERR: Cannot clone request"))?
                     .send()
                     .await?;
+                trace!("{:?}", response);
```

Comment:
this is in trace mode. Could you change it to debug, and add a `info` only for the http status code?

Reply:
status should not be in info status unless it is not 200